modules/array.js

```diff
@@ -53,33 +52,3 @@ export const code = {
   $: "...-..-",
   _: "..--.-",
 };
```

Comment:
array.js is not a descriptive enough name for this file. 
Could you think more of the actual content of the array in this file rather than the data structure in it?
You will have many files with many arrays in them. So it would be best if you were more specific.
dictionary.js  would be a more appropriate name in this case

Reply:
I see what you're saying, new name is morse-english-dictionary.js